modules/dynamic-plug-in-sdk-extensions.adoc

```diff
@@ -82,19 +82,23 @@ which returns actions for the given resource model
 [discrete]
 == `console.alert-action`
 
+This extension can be used to trigger a specific action when a specific Prometheus alert is observed by the Console based on its `rule.name` value.
+
 [cols=",,,",options="header",]
 |===
 |Name |Value Type |Optional |Description
-|`alert` |`string` |no |
+|`alert` |`string` |no | Alert name as defined by `alert.rule.name` property |
 
-|`text` |`string` |no |
+|`text` |`string` |no |  Action text |
 
-|`action` |`CodeRef<(alert: any) => void>` |no |
+|`action` |`CodeRef<(alert: any) => void>` |no | Function to perform side effect |
```

Comment:
it looks like we have some formatting issue in the preview
![Screenshot 2023-03-30 at 2 02 07 PM](https://user-images.githubusercontent.com/12692381/228743737-31c679d4-bd17-4b53-a2d5-fb3faf892ef3.png)